### ISSUE TITLE
support for sass

### ISSUE THREAD

**Issue by ghost** [Maushundb/quick-editor#1]
add support for SASS pls

**Comment by benadamstyles** [comment]
Don't know if this is useful but I'll register my desire for SASS support too – :+1:!

**Comment by Maushundb** [comment]
Yeah this is definitely something I want to do...but the syntax is a little trickier to parse and would take quite a bit of refactoring of some things (my fault really for not thinking this far ahead) so its somewhat on the back burner for the time being. I'm currently focusing on improving functionality for currently supported file types. 

I'll post here again when I start working on it!

**Comment by dahliacreative** [comment]
Does anyone actually use Less or Scss? 👍  for Sass support!

**Comment by dahliacreative** [comment]
On a side note, I use bem so all my filenames match the css class names I use, eg `class="card-widget` would be in `card-widget.sass` how hard would it be to just get the quick editor to open up the corresponding file and if one doesn't exist give the option to create it?

I've never done any type of editor plugin before, but if it was relatively simple I could branch and have a bash, don't know how far I would get as from what I've seen looks a bit over my head.
